test/e2e/README.md

```diff
@@ -0,0 +1,91 @@
+# End to End Testing with Package-Feeds integration
+
+This directory helps run end-to-end tests of the package analysis system
+to ensure everything is working properly.
+In particular, local changes to both the worker/analysis and sandbox images can be tested
+before they are pushed to the docker registry.
+
+The test is orchestrated using docker-compose, using a setup adapted from the end-to-end
+example under `examples/e2e`. However, all the commands can be run via the project Makefile.
```

Comment:
Is this still accurate? It's under "config/" right?

Reply:
The README is still in `examples/e2e` and technically it still represents an example... should I move the readme to config too?

Reply:
The confusing bit to me is the "using a setup adapted from" bit because the bulk of the config got moved to `/configs` right? Might be worth just rewording this sentence to

"based on the base setup from /configs/.."

Reply:
Yeah that's a better idea, thanks